storops/unity/resource/pool.py

```diff
@@ -203,6 +206,51 @@ def disk_groups(self):
                 dgs[pd.disk_group.get_id()] = [pd]
         return dgs
 
+    @version('<5.1')
+    def is_compression_enabled(self):
+        return self.is_all_flash
+
+    @version('>=5.1')  # noqa
+    def is_compression_enabled(self):
+        resp = self._cli.action(self.resource_class, self.get_id(),
+                                'verifyDataReduction')
+        resp.raise_if_err()
+        if resp.body.get('content') and resp.body.get('content').get(
+                'verifyPass') is True:
```

Comment:
`is True` is redundant.
You could also `return resp.body.get('content') and resp.body.get('content').get('verifyPass')`.

Reply:
Done.